src/components/forms/SubnetTierForm.js

```diff
@@ -6,19 +6,24 @@ import {
   IcseSubForm,
   StatelessToggleForm,
 } from "../Utils";
-import { titleCase } from "lazy-z";
+import { buildNumberDropdownList } from "lazy-z";
 import PropTypes from "prop-types";
 import { DeleteButton, SaveAddButton } from "../Buttons";
 import { IcseNameInput, IcseToggle } from "../Inputs";
 import { IcseNumberSelect, IcseSelect } from "../Dropdowns";
 import SubnetTileForm from "./SubnetTileForm";
 import { subnetTierName } from "../../lib";
-import { contains } from "lazy-z";
+import { IcseMultiSelect } from "../MultiSelects";
 
 class SubnetTierForm extends React.Component {
   constructor(props) {
     super(props);
     this.state = { ...this.props.data };
+    if (!this.state.select_zones) {
+      let zones = buildNumberDropdownList(this.state.zones, 1);
+      this.state.select_zones = [];
+      zones.forEach((zone) => this.state.select_zones.push(Number(zone)));
+    }
```

Comment:
zones are displaying [1, 3, 2] - can we make them in order?

Reply:
i think it defaults to putting the selected ones first but i'll try

Reply:
yeah this is not possible